sdks/python/apache_beam/testing/load_tests/build.gradle

```diff
@@ -29,18 +29,20 @@ def loadTestArgsProperty = "loadTest.args"
 
 def runnerProperty = "runner"
 
-task run(type: Exec) {
-    dependsOn 'setupVirtualenv'
-    dependsOn 'installGcpTest'
-
+task run(type: Exec, dependsOn: installGcpTest) {
     def loadTestArgs = project.findProperty(loadTestArgsProperty) ?: ""
 
     def runnerArg = project.findProperty(runnerProperty) ?: ""
     if (runnerArg == 'DataflowRunner' || runnerArg == 'TestDataflowRunner') {
-        dependsOn 'sdist'
-        loadTestArgs +=" --sdk_location=${project.buildDir}/apache-beam.tar.gz"
+        def sdistPath = "${project.rootDir}/sdks/python/apache_beam/testing/load_tests/build/apache-beam.tar.gz"
```

Comment:
Is this using a different path than `${project.buildDir}/apache-beam.tar.gz` (I don't know what buildDir expands to)?
Is this a custom tar.gz file?

Reply:
Well, this is caused by recent changes in `BeamModulePlugin` - `sdist` task has `dist-dir` pointed to `project.buildDir`. Because load-tests gradle file is located in load tests dir, the buildDir is location of load tests. So it creates package there.

Well, one of suggestion in past was to move this task to other tasks in main `build.gradle`. But I postponed decision.

Reply:
It seems that if `sdist` creates the file `${sdistPath}`, then `dependsOn "sdist"` should always be used (no need for `if(!fileExists(sdistPath))`).

Reply:
Those load tests are run 4 times in various combinations (plan is to run more). I believe `sdist` is quite time-consuming operation I wanted to run it only once during whole tests suite. WDYT about it?

Reply:
Gradle is smart about task dependencies. As long as the input files to `sdist` don't change it should run it only once.

Reply:
When I removed it, jenkins job makes sdist 4 times :( [here](https://builds.apache.org/job/beam_Python_LoadTests_GBK_Dataflow_Batch_PR/11/console#gradle-task-24)

Reply:
That's a bug. Please let it run 4 times for now and file an issue in JIRA.

Reply:
Ok, good to know this is something worth reporting. I created an [issue](https://issues.apache.org/jira/browse/BEAM-7432) and I'm leaving it as it is.

Reply:
@udim is there something more to be changed here?